### PR TITLE
Mul 138 140

### PR DESCRIPTION
**Pull request by Anschutz1927**
@AndreyGrowl @ihar-paliashchuk please review

### CODE REVIEW

File: app/src/main/java/io/multy/ui/activities/MainActivity.java [hunk 1]
```diff
@@ -65,21 +74,21 @@ public void onBackPressed() {
...
+                setFragment(R.id.inner_container, FeedFragment.newInstance());
                 break;
             case Constants.POSITION_CONTACTS:
-//                setFragment(R.id.full_container, ContactsFragment.newInstance());
-                startActivity(new Intent(this, AssetSendActivity.class));
+                setFragment(R.id.inner_container, ContactsFragment.newInstance());
                 break;
             case Constants.POSITION_SETTINGS:
-//                setFragment(R.id.full_container, SettingsFragment.newInstance());
+                setFragment(R.id.inner_container, SettingsFragment.newInstance());
```
Comment (AndreyGrowl) [inline app/src/main/java/io/multy/ui/activities/MainActivity.java]: use naming like container_what. eg - container_frame not where or how (inner)

File: app/src/main/java/io/multy/ui/adapters/PortfoliosAdapter.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+/**
+ * Created by appscrunch on 16.11.17.
+ */
+
+public class PortfoliosAdapter extends PagerAdapter {
+
+
+    @Override
+    public int getCount() {
+        return 10;
```
Comment (AndreyGrowl) [inline app/src/main/java/io/multy/ui/adapters/PortfoliosAdapter.java]: Always define constant in Class' fields -> public static final int COUNT = 10;

File: app/src/main/java/io/multy/ui/adapters/PortfoliosAdapter.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    @Override
+    public void destroyItem(ViewGroup container, int position, Object object) {
+        container.removeView((View) object);
+    }
+
+    @Override
+    public CharSequence getPageTitle(int position) {
+//        return super.getPageTitle(position);
+        return "Portfolio " + String.valueOf(position);
```
Comment (AndreyGrowl) [inline app/src/main/java/io/multy/ui/adapters/PortfoliosAdapter.java]: Don't use hardcoded strings

File: app/src/main/java/io/multy/ui/adapters/WalletsAdapter.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    @Override
+    public Holder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View v = LayoutInflater.from(parent.getContext())
+                .inflate(R.layout.view_asset_item, parent, false);
+        return new Holder(v);
+    }
+
+    @Override
+    public void onBindViewHolder(Holder holder, int position) {
+//        holder.name.setText(data.get(position).name);
```
Comment (AndreyGrowl) [inline app/src/main/java/io/multy/ui/adapters/WalletsAdapter.java]: actually, we don't need so much commented code in our repository

File: app/src/main/java/io/multy/ui/fragments/asset/AssetInfoFragment.java [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+import io.multy.ui.fragments.BaseFragment;
+
+public class AssetInfoFragment extends BaseFragment {
+
+    public static final String TAG = AssetInfoFragment.class.getSimpleName();
+
+    @BindView(R.id.collapsing_toolbar)
+    CollapsingToolbarLayout collapsingToolbarLayout;
+    @BindView(R.id.recycler_transactions)
+    RecyclerView recyclerTransactions;
```
Comment (AndreyGrowl) [inline app/src/main/java/io/multy/ui/fragments/asset/AssetInfoFragment.java]: if recyclerView is only one in class, call it just recyclerView

File: app/src/main/res/layout/view_transaction_item.xml [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+        tools:text="0.052509 BTC" />
+
+    <TextView
+        android:id="@+id/text_time"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:fontFamily="@font/montserrat"
+        android:maxLines="1"
+        android:text="200.34 USD"
```
Comment (ihar-paliashchuk) [inline app/src/main/res/layout/view_transaction_item.xml]: Move to resources

**Pull request merged**